templates/formation/_navbar.html.twig

```diff
@@ -1,19 +1,33 @@
-<nav>
+<nav class="espace-formation__nav space--20-0 l__row">
     <ul class="list__links list__links--row list__links--no-decor list__links--nav">
         <li>
             <a href="{{ path('app_formation_home') }}" class="{{ is_active_route(app.request, 'app_formation_home') ? 'active' }}">
                 Mon parcours
             </a>
         </li>
-        <li>
-            <a href="{{ path('app_formation_faq') }}" class="{{ is_active_route(app.request, 'app_formation_faq') ? 'active' }}">
-                Foire aux questions
-            </a>
-        </li>
-        <li>
-            <a href="#">
-                Faire le QCM
-            </a>
-        </li>
+        {% if app.request.get('_route') == 'app_formation_home' %}
```

Comment:
Here we need something like `{% if not app.request.get('_route') == 'app_formation_article' %}`but i couldnt make it works

Reply:
you can use this: 
```
{% if is_active_route(app.request, 'app_formation_home') %}
```